app/PrintJob.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PrintJob extends Model
+{
+    protected $table = 'print_jobs';
+    protected $primaryKey = 'id';
+    public $incrementing = true;
```

Comment:
What does this field mean?

Reply:
`$incrementing` is a Laravel-specific variable, telling Laravel that the primary key is auto-incrementing.